primefaces-integration-tests/src/main/webapp/datepicker/datePicker012.xhtml

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:h="http://xmlns.jcp.org/jsf/html"
+      xmlns:f="http://xmlns.jcp.org/jsf/core"
+      xmlns:p="http://primefaces.org/ui">
+
+<f:view contentType="text/html;charset=UTF-8" encoding="UTF-8">
+    <h:head>
+
+    </h:head>
+
+    <h:body>
+
+        <h:form id="form">
+            <p:datePicker id="datepicker1" value="#{datePicker012.date1}"
+                          model="#{datePicker012.lazyModel1}" refreshLazyModel="true"
```

Comment:
refreshLazyModel="true"? There is no such attribute.

Reply:
Agreed there is no property for that...